src/components/sidebar/Sidebar.vue

```diff
@@ -427,6 +427,20 @@ export default {
       saveAs(jsonldFile, `${this.downloadFilenameForPhyx}.jsonld`);
     },
 
+    downloadAsNQuads() {
+      // Exports the PHYX file as an OWL/N-Quads file, which can be opened in
+      // Protege or converted into other RDF formats.
+      const wrapped = new PhyxWrapper(this.$store.state.phyx.currentPhyx);
+
+      // TODO: we need a baseIRI here because of https://github.com/phyloref/phyx.js/issues/113
+      // Once that is fixed in phyx.js, we can remove it here.
+      wrapped.toRDF('https://example.org/phyx#').then((content) => {
```

Comment:
For aesthetic reasons, wouldn't something more personalized be preferable, such as `https://example.phyloref.org/phyx#`?

Reply:
Sure! I always prefer `example.org` since it's widely known to be non-resolvable, but the actual value doesn't really matter. I've changed it to the IRI you suggest in cd0e4e0.